lib/mix/tasks/gen.user_from_csv.ex

```diff
@@ -0,0 +1,103 @@
+defmodule Mix.Tasks.Gen.UserFromCsv do
+  use Mix.Task
+  alias Ecto.Multi
+  alias Safira.Repo
+  alias Safira.Auth
+  alias Safira.Accounts.User
+  alias Safira.Accounts.Attendee
+
+  NimbleCSV.define(SeiParser, separator: ",", escape: "\"")
```

Comment:
If you are using a RFC4180 csv compliant file, you can use the `NimbleCSV.RFC4180` instead.
[https://hexdocs.pm/nimble_csv/NimbleCSV.html#module-parsing](https://hexdocs.pm/nimble_csv/NimbleCSV.html#module-parsing)